app/javascript/styles/fonts/montserrat.scss

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
It isn't too weird really, as some React practices involve bundling CSS modules directly with components, e.g. CSS and JS side by side, with any relevant images included. We don't use that but it's still all close together.